doc/kubernetes/modules/ROOT/pages/customizing-deployment.adoc

```diff
@@ -131,6 +131,43 @@ When the image name starts with `localhost/`, the image will be expected to be b
 +
 See xref:util/custom-image-for-keycloak.adoc[] on how to build a local image.
 
+KC_INSTANCES::
+Sets the number of Keycloak instances.
++
+Default value: `1`
+
+KC_CPU_LIMITS::
```

Comment:
I'm about to add changes to fix OpenTelemetry. To avoid conflicts, I'm merging this PR. 

@tkyjovsk - please create another PR for the additional docs. Thanks!